src/BTVNanoCommissioning/helpers/definitions.py

```diff
@@ -6277,27 +6280,39 @@ def axes_name(var_input):
         elif "btagDeepCvB" in var:
             unit = "Jet DeepCSV CvB"
         elif "btagDeepB" in var:
-            unit = "Jet DeepCSV Prob(b)"
+            unit = "Jet DeepCSV Prob (b)"
         elif "btagDeepC" in var:
-            unit = "Jet DeepCSV Prob(c)"
+            unit = "Jet DeepCSV Prob (c)"
         elif "btagDeepFlavCvL" in var:
             unit = "Jet DeepJet CvL"
         elif "btagDeepFlavCvB" in var:
             unit = "Jet DeepJet CvB"
         elif "btagDeepFlavB" in var:
-            unit = "Jet DeepJet Prob(b)"
+            unit = "Jet DeepJet Prob (b)"
         elif "btagDeepFlavC" in var:
-            unit = "Jet DeepJet Prob(c)"
+            unit = "Jet DeepJet Prob (c)"
```

Comment:
Here I see (potentially) a general problem, namely that (PF)Nano lists not only bTagDeepB, but also things like bTagDeepB_b, _bb (similar for DeepFlav). The problem is that you write Prob (b), which is, strictly speaking, only correct for one of the three (or four) `var` that match the pattern. The one you probably meant here was the sum of P(b)+P(bb) for DeepCSV or similar sum + P(lepb) for DeepJet. So you need to check later, that the actual P(b) and P(bb) (and P(lepb)) get their own labels (units, as they are called here). Note that this does not occur for bTagDeep(Flav)C, because c is not split further in any of the two currently investigated taggers here, i.e. Prob(c) would be sth like CvsAll.
For the sum of the two or three individual b jet categories, one would have to write something that does not contain "Prob", examples are just a capital B or b discriminator etc.
On a similar note, we might actually be missing the individual output nodes in this set of definitions, though they are probably not the most important thing to add, the individual probabilities sometimes give us additional information to understand where disagreement is coming from. Even if we don't include them, the current label does not cover the fact that it's a sum of probabilities for b / bb (/ lepb).

Reply:
Hi @AnnikaStein thx for your comment! I have changed the axis labels and added the P(b), P(bb) and P(lepb) vars as well (hence the change in `histogrammer.py`)

Reply:
Yes, indeed we need add condition [here](https://github.com/cms-btv-pog/BTVNanoCommissioning/blob/master/src/BTVNanoCommissioning/workflows/ttsemilep_valid_sf.py#L350-L354) just to skip the the SFs part

Reply:
So smth like:
```
                        if (
                            not isRealData
                            and self.isCorr
                            and "BTV" in correction_config[self._campaign].keys()
                            and "_b" not in histname
                            and "_bb" not in histname
                            and "_lepb" not in histname
                        ):
```
will probably do the job. I will quickly test it